pages/index.js

```diff
@@ -190,6 +301,9 @@ export default function Homepage() {
         <ThemeToggle />
         <BackToTop />
       </main>
+      {renderHistoryModal(searchHistory, showHistoryModal, () => {
```

Comment:
@icegotcha I think there are too many things in the nav...and would prefer if history was hidden _until_ I click the history button.

Reply:
Done. I removed the third line in the nav and place the button next to example search terms.